app/subscriber/src/features/search-page/components/advanced-search/constants/DefaultAdvancedSearch.ts

```diff
@@ -13,8 +13,9 @@ export const defaultAdvancedSearch = {
   page: '',
   publishedEndOn: '',
   publishedStartOn: '',
-  search: '',
+  // search: '', // don't reset this...
```

Comment:
Won't this cause a undefined error on initial load?  Why not just override this when reseting?

Reply:
changed as per suggestion